pkg/controller/node/nodecontroller.go

```diff
@@ -314,6 +314,9 @@ func NewNodeController(
 	})
 	nc.podInformerSynced = podInformer.Informer().HasSynced
 
+	addNodeFunctions := []func(obj *v1.Node) error{}
```

Comment:
As discussed offline, we should have multiple "AddEventHandlers" functions instead of this.